DS-Algo-Projects/RBMazeGen.java

```diff
@@ -152,20 +152,23 @@ public void paint(Graphics g2) {
             g.setStroke(new BasicStroke(2));
             for(int i = 1; i < w ; i+=RectSize){
                 for(int j = 1; j < h ; j+=RectSize){
-                    if(currently.contains(arr[i/RectSize][j/RectSize])) {
+
+                    Node current = arr[i/RectSize][j/RectSize];
+
+                    if(currently.contains(current)) {
                         g.setColor(Color.green);
```

Comment:
Hi, can you please go through once again and comment on what the variables mean and their usage. 
Thanks.

Reply:
Done, have a look if you need me to add something more.

Reply:
sample I/O and complexity if needed. Have a look at other codes in the repo.

Reply:
bruh? what will be input for maze generating algorithm.

Reply:
makes sense, I been switching between code reviews, apologies.

Reply:
Yeah, thank you anyway